libs-scala/oracle-testing/src/main/scala/testing/oracle/OracleAround.scala

```diff
@@ -40,7 +40,7 @@ trait OracleAround {
       val con = use(
         DriverManager.getConnection(
           s"jdbc:oracle:thin:@localhost:$port/ORCLPDB1",
-          systemUser,
+          "sys as sysdba", // TODO this is needed for being able to grant the execute access for the sys.dbms_lock below. Consider making this configurable
```

Comment:
@dasormeter can you please comment on this?

Reply:
we need the extra permissions if we are making use of oracle locking - it's actually a good point that this may not be needed for index server as that locking functionality is only there for canton, and i don't think is applied for the index server

Reply:
alright, I leave it like this, and extend `DPP-495 Unit testing: DBLockStorageBackend` with sorting this out with the option/flag.